### ISSUE TITLE
unable to do npm run dev

### ISSUE THREAD

**Issue by ThekhoN** [ezekielchentnik/preact-pwa#10]
i cloned your repository and installed the dependencies.
however when i run yarn/npm run dev, i get this error:
```
[nodemon] starting `'node build`
''node' is not recognized as an internal or external command,
operable program or batch file.
[nodemon] app crashed - waiting for file changes before starting...

```
Can you check if the package.json is correct?

Thanks.

**Comment by ezekielchentnik** [comment]
@ThekhoN are you on win/mac?  I just merged #5 , I believe this will resolve your issue.  Please grab latest and give it a try, let me know how it goes.

Cheers

**Comment by ThekhoN** [comment]
Awesome. it's working now. thank you :)
